docs/DeveloperGuide.adoc

```diff
@@ -27,15 +27,15 @@
 . Add JUnit 4 to classpath
 .. Open any test file in `\test\java` and place your cursor over any `@Test` highlighted in red
 .. Press ALT+ENTER and select `Add 'JUnit4' to classpath`
-.. Select `Use 'JUnit4' from IntelliJ IDEA distribution` and click `OK`
+.. Select `junit:junit:4.12` and click `OK`
```

Comment:
I wonder if we should future-proof this line a bit by saying `if 4.12 is not available, choose any 4.x that is later than 4.12` 🤔

Reply:
Sounds good 👍.